.env

```diff
@@ -0,0 +1,3 @@
+API_KEY=
```

Comment:
the .env file, should be ignored, and not in the repo

Reply:
Yes, I write a message for this file in discord